rfc.md

```diff
@@ -179,10 +160,73 @@ class Spades extends Suit {
 }
 ```
 
+### Value listing
+
+The enumeration itself has an automatically generated static method `values()`.  `values()` returns an indexed array of all defined Cases in lexical order.
```

Comment:
I've never heard them called "packed" arrays outside of internals discussions.  "indexed array" and "associative array" are the standard terms I see just about everywhere except in engine discussions.

Reply:
"Packed" in this context means starting with index 0 and always incrementing by 1, so what other languages would consider normal arrays.

Reply:
Eh.  I guess for an RFC calling it that may be more precise, but no one who doesn't read Internals uses that term.  Most have probably never even heard of it.

Reply:
So, just drop the indexed part altogether then. All arrays in PHP are indexed.

Reply:
I already updated it to packed.  But it may be a language difference here.  I've never seen an English speaker use "packed array" instead of "indexed array' when referring to a numeric-key array.